flux_led/__main__.py

```diff
@@ -661,28 +661,41 @@ def getRgbw(self):
         return (red, green, blue, white)
 
     def setRgbw(self, r,g,b,w, persist=True, brightness=None, retry=2):
+        if (r or g or b) and w and not self.rgbwcapable:
+            print("RGBW command sent to non-RGBW device")
+            raise Exception
+
         if brightness != None:
             (r, g, b) = self._calculateBrightness((r, g, b), brightness)
+
         if persist:
             msg = bytearray([0x31])
         else:
             msg = bytearray([0x41])
+
         msg.append(int(r))
         msg.append(int(g))
         msg.append(int(b))
         msg.append(int(w))
-
         if self.protocol == "LEDENET":
             msg.append(0x00)
```

Comment:
Should probably add 
```
       if level > 0
           msg.append(int(level))
      else:
           msg.append(0x00)
```
https://github.com/Danielhiversen/flux_led/pull/23/files#diff-432c4d3e79417f8676dc71ae5e419702L643